modules/table/src/main/java/org/apache/ignite/internal/table/distributed/storage/InternalTableImpl.java

```diff
@@ -1041,7 +1074,8 @@ public CompletableFuture<List<BinaryRow>> insertAll(Collection<BinaryRowEx> rows
 
                     // All values are null, this means nothing was deleted.
                     return true;
-                }
+                },
+                false
```

Comment:
could you pls clarify why dont we need auto retry here (and in deleteAll), while we have it for upsertAll?

Reply:
Currently retries only enabled for a datastreamer as a showcase. All other scenarios must be evaluated in different ticket due to a possible impact on test logic.